src/packages/engine-core/src/NodeEngine.ts

```diff
@@ -194,17 +198,18 @@ export class NodeEngine implements Engine {
         )}\` were removed, you can now safely remove them from your schema.prisma.`,
       )
     }
-
     this.previewFeatures = this.previewFeatures.filter(
       (e) => !removedFlags.includes(e),
     )
+    
+    // custom endpoint
     this.engineEndpoint = engineEndpoint
-
     if (engineEndpoint) {
       const url = new URL(engineEndpoint)
       this.port = Number(url.port)
     }
 
+    // TODO
```

Comment:
What is this exactly? I have trouble understanding what the error message really tells me via this conditional.

Reply:
```suggestion
    // TODO What is this exactly? I have trouble understanding what the error message really tells me via this conditional.
```